fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/api/transfer/v1/context/ContainerItemContext.java

```diff
@@ -24,6 +24,7 @@
 import net.minecraft.entity.player.PlayerEntity;
 import net.minecraft.entity.player.PlayerInventory;
 import net.minecraft.item.ItemStack;
+import net.minecraft.item.ItemUsage;
```

Comment:
Hmm, do we generally do javadoc-only imports? Feels a bit weird to me to only reference imported classes in comments

Reply:
I do it to avoid rendering the fully qualified name: (and it's done throughout the transfer API already)
![image](https://user-images.githubusercontent.com/13494793/210411250-0b63fc1e-a020-4143-867f-acccfe0fabc6.png)